jdk/test/sun/management/jmxremote/startstop/JMXStartStopTest.java

```diff
@@ -702,29 +749,30 @@ static void test_10() throws Exception {
         // make sure these properties overridden corectly
 
         System.out.println("**** Test ten ****");
+        PortAllocator pa = new PortAllocator();
 
         Something s = doSomething(
             "test_10",
-            "-Dcom.sun.management.jmxremote.port=" + port1,
+            "-Dcom.sun.management.jmxremote.port=" + pa.getPort1(),
             "-Dcom.sun.management.jmxremote.authenticate=false",
             "-Dcom.sun.management.jmxremote.ssl=true");
 
         try {
-            testNoConnect(port1);
+            testNoConnect(pa.getPort1());
 
             jcmd(CMD_STOP);
             jcmd(CMD_START,
                 "jmxremote.ssl=false",
-                "jmxremote.port=" + port1
+                "jmxremote.port=" + pa.getPort1()
             );
             testConnect(port1);
```

Comment:
fyi: change to port1 is missed, see https://github.com/openjdk/jdk8u-dev/pull/429